Components/Widgets/Container.js

```diff
@@ -16,6 +16,23 @@ export default class Container extends NativeBaseComponent {
 	propTypes: {
         style : React.PropTypes.object
     }
+	
+	renderOther(){
+		if(Array.isArray(this.props.children)) {
+
+			return _.filter(this.props.children, function(item) {
+				if(item && (item.type != Header && item.type != ViewNB && item.type != Content && item.type != Image && item.type != View && item.type != ScrollView && item.type != Footer)) {
```

Comment:
You can use _contains from lodash, e.g. `_.contains([1, 2, 3], 3);` returns true